content/guides/models/registry/lineage.md

```diff
@@ -0,0 +1,94 @@
+---
+description: Create a lineage map in the W&B Registry.
+menu:
+  default:
+    parent: registry
+    identifier: lineage
+title: Create and view lineage maps
+weight: 8
+---
+
+Within a collection in the W&B Registry, you can view a history of the artifacts that were created and used throughout an ML experiment. This history is called a _lineage graph_.
+
+{{% alert %}}
+You can also view lineage graphs for artifacts you log to W&B that are not part of a collection.
```

Comment:
This could be demoted from an alert and moved into the previous paragraph, I think. "an ML experiment or an artifact uses" perhaps?